vcd/resource_vcd_vapp_vm.go

```diff
@@ -441,7 +445,18 @@ func resourceVcdVAppVmUpdate(d *schema.ResourceData, meta interface{}) error {
 
 		if d.HasChange("cpus") {
 			err = retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-				task, err := vm.ChangeCPUCount(d.Get("cpus").(int))
+				var task govcd.Task
+				var err error
+				if d.HasChange("cpu_cores") {
```

Comment:
We won't get here if someone changes just 'cpu_cores' without altering 'cpus'

Reply:
Fixed, though it worked somehow.